Apps/WebClient/src/ClientApp/src/components/timeline/medication.vue

```diff
@@ -194,6 +199,10 @@ $radius: 15px;
                         </b-collapse>
                     </b-col>
                 </b-row>
+            </b-col>
+        </b-row>
+        <b-row>
```

Comment:
Looks like all implementation of the comment section live inside a row and column. If that is the case lets move both to the comment section itself as well as the style that is only being used there

Reply:
Thanks @FuriousLlama ! I still prefer the rounded corners with padding left, right, bottom version. Can we get the inputs from @ninosamson and team to confirm?

Reply:
Sure thing
Expected:
![image](https://user-images.githubusercontent.com/12201292/86155782-be247d80-bab9-11ea-8857-603945066f69.png)

Reply:
Great, thanks @FuriousLlama. Per our team's decision today, we will go ahead with @FuriousLlama 's suggestion.